[misc] Reimport cardiac rehab questionnaires

acrowthe opened the pull request:
- Fix question options name

acrowthe committed:
[misc] Reimport cardiac rehab questionnaires
- Fix question options name

marta- commented on cardiac-rehab-resources/clinical-data/src/main/resources/SLING-INF/content/Questionnaires/6MWT.xml:
```diff
@@ -771,11 +771,6 @@
 				<value>1</value>
 				<type>Long</type>
 			</property>
-			<property>
-				<name>maxAnswers</name>
-				<value>1</value>
-				<type>Long</type>
-			</property>
```
Update the spreadsheet

marta- commented on cardiac-rehab-resources/clinical-data/src/main/resources/SLING-INF/content/Questionnaires/6MWT.xml:
```diff
@@ -800,11 +795,6 @@
 				<value>1</value>
 				<type>Long</type>
 			</property>
-			<property>
-				<name>maxAnswers</name>
-				<value>1</value>
-				<type>Long</type>
-			</property>
```
Update the spreadsheet

marta- commented on cardiac-rehab-resources/clinical-data/src/main/resources/SLING-INF/content/Questionnaires/6MWT.xml:
```diff
@@ -829,11 +819,6 @@
 				<value>1</value>
 				<type>Long</type>
 			</property>
-			<property>
-				<name>maxAnswers</name>
-				<value>1</value>
-				<type>Long</type>
-			</property>
```
Update the spreadsheet

marta- commented on cardiac-rehab-resources/clinical-data/src/main/resources/SLING-INF/content/Questionnaires/6MWT.xml:
```diff
@@ -843,6 +828,11 @@
 				<value>Total distance</value>
 				<type>String</type>
 			</property>
+			<property>
+				<name>description</name>
+				<value>m</value>
+				<type>String</type>
+			</property>
```
Update the spreadsheet

marta- commented on cardiac-rehab-resources/clinical-data/src/main/resources/SLING-INF/content/Questionnaires/6MWT.xml:
```diff
@@ -858,11 +848,6 @@
 				<value>1</value>
 				<type>Long</type>
 			</property>
-			<property>
-				<name>maxAnswers</name>
-				<value>1</value>
-				<type>Long</type>
-			</property>
```
Update the spreadsheet

marta- commented on cardiac-rehab-resources/clinical-data/src/main/resources/SLING-INF/content/Questionnaires/Cardiac Rehab Exercise Diary.xml:
```diff
@@ -341,24 +341,14 @@
 			<property>
 				<name>dataType</name>
-				<value>decimal</value>
+				<value>long</value>
```
Update the spreadsheet

marta- commented on cardiac-rehab-resources/clinical-data/src/main/resources/SLING-INF/content/Questionnaires/Cardiac Rehab Exercise Diary.xml:
```diff
@@ -341,24 +341,14 @@
 				<value>m</value>
 				<type>String</type>
 			</property>
-			<property>
-				<name>minAnswers</name>
-				<value>0</value>
-				<type>Long</type>
-			</property>
-			<property>
-				<name>maxAnswers</name>
-				<value>1</value>
-				<type>Long</type>
-			</property>
```
Update the spreadsheet

marta- commented on cardiac-rehab-resources/clinical-data/src/main/resources/SLING-INF/content/Questionnaires/Participant Status.xml:
```diff
@@ -375,11 +375,6 @@
 				<value>1</value>
 				<type>Long</type>
 			</property>
-			<property>
-				<name>maxAnswers</name>
-				<value>1</value>
-				<type>Long</type>
-			</property>
```
Update the spreadsheet

marta- commented on cardiac-rehab-resources/clinical-data/src/main/resources/SLING-INF/content/Questionnaires/Participant Status.xml:
```diff
@@ -391,16 +386,16 @@
-				<value>Year of birth</value>
+				<value>Date of birth</value>
 				<type>String</type>
 			</property>
 			<property>
 				<name>description</name>
-				<value>Format: YYYY, for example 1950</value>
+				<value>year only</value>
```
Update the spreadsheet

marta- commented on cardiac-rehab-resources/clinical-data/src/main/resources/SLING-INF/content/Questionnaires/6MWT.xml:
```diff
@@ -27,7 +27,7 @@
 	<property>
 		<name>description</name>
-		<value>Visits 2, 4, 7, 8</value>
+		<value>Visits 2,4,7,8</value>
```
Update the spreadsheet

marta- commented on cardiac-rehab-resources/clinical-data/src/main/resources/SLING-INF/content/Questionnaires/6MWT.xml:
```diff
@@ -57,7 +57,7 @@
 		<property>
 			<name>description</name>
-			<value>2, 4, 7, 8</value>
+			<value>1-8</value>
```
Update the spreadsheet

marta- commented on cardiac-rehab-resources/clinical-data/src/main/resources/SLING-INF/content/Questionnaires/Baseline Health Information - Cardiovascular parameters.xml:
```diff
@@ -65,7 +65,7 @@
 			<property>
 				<name>description</name>
-				<value>1, 5, 6, 8</value>
+				<value>1-8</value>
```
Update the spreadsheet

marta- commented on cardiac-rehab-resources/clinical-data/src/main/resources/SLING-INF/content/Questionnaires/Baseline Health Information - Medications.xml:
```diff
@@ -65,7 +65,7 @@
 			<property>
 				<name>description</name>
-				<value>1, 5, 6, 8</value>
+				<value>1-8</value>
```
Update the spreadsheet

marta- commented on cardiac-rehab-resources/clinical-data/src/main/resources/SLING-INF/content/Questionnaires/Daily diary.xml:
```diff
@@ -140,11 +140,6 @@
 				<value>long</value>
 				<type>String</type>
 			</property>
-			<property>
-				<name>unitOfMeasurement</name>
-				<value>n</value>
-				<type>String</type>
-			</property>
```
This change is fine

marta- reviewed (changes requested):
Before testing begins, the spreadsheet needs to be updated with the changes from the code base (as marked below in individual comments), and the forms re-imported.

sashaandjic commented:
small inconsistency - missing spaces
![image](https://user-images.githubusercontent.com/23243244/102527831-b9558500-406b-11eb-9816-382c391ebfc0.png)

acrowthe committed:
[misc] Reimport cardiac rehab questionnaires
- Update spreadsheet to undo unintended changes

acrowthe committed:
[misc] Reimport cardiac rehab questionnaires
- Adjust maxAnswers ordering for consistency

acrowthe committed:
[misc] Reimport cardiac rehab questionnaires
Fix year_of_birth node name

marta- commented on cardiac-rehab-resources/clinical-data/src/main/resources/SLING-INF/content/Questionnaires/Cardiac Rehab Exercise Diary.xml:
```diff
@@ -351,7 +351,7 @@
 			<property>
 				<name>minAnswers</name>
-				<value>0</value>
+				<value>1</value>
```
Update the spreadsheet

marta- commented on cardiac-rehab-resources/clinical-data/src/main/resources/SLING-INF/content/Questionnaires/Surveys.xml:
```diff
@@ -65,7 +65,7 @@
 			<property>
 				<name>description</name>
-				<value>1, 8</value>
+				<value>1-8</value>
```
Update the spreadsheet

acrowthe committed:
[misc] Reimport cardiac rehab questionnaires
- Clean up last two unintended changes

marta- reviewed (approved):
All looks right to me. didn't test

sashaandjic reviewed (approved):
tested
Defect described in [LFS-761](https://phenotips.atlassian.net/browse/LFS-761) is still happening

acrowthe committed:
[misc] Reimport cardiac rehab questionnaires

Fix issue reported in [LFS-761](https://phenotips.atlassian.net/browse/LFS-761)

acrowthe commented:
Fixed LFS-761 issue for now: @k-pace is determining what correct behavior is for those two medication information sections, so another change there may be required

sashaandjic commented:
> Fixed LFS-761 issue for now: @k-pace is determining what correct behavior is for those two medication information sections, so another change there may be required

Not good. `Medication2` fields are only showing if you enter 3 in field `How many medications are prescribed?`
